week2_PS_draft.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<p>Rename file name to current name in your directory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>